giskard/rag/question_generators/ookb_questions.py

```diff
@@ -0,0 +1,176 @@
+import uuid
+
+from ..knowledge_base import KnowledgeBase
+from .base import GenerateFromSingleQuestionMixin, _LLMBasedQuestionGenerator
+from .prompt import QAGenerationPrompt
+
+OOKB_PROMPT = """
+You are a powerful auditor and mindful judger, your role is to generate question from a given context and 
+by adding some fake or non-existing details/facts to the context to check whether the agent model you are auditing is capable of reasoning and answering questions
+which has no direct answer in the provided context.
+
+There are your tasks, you should finish them step by step:
+0. select one detail/fact from the context
```

Comment:
Minor stuff: I would index the list beginning with 1 instead of 0, it's more common.